java/src/main/java/io/grpc/examples/wallet/AccountServer.java

```diff
@@ -97,17 +110,30 @@ private void start() throws IOException {
       Observability.registerExporters(gcpClientProject);
     }
     HealthStatusManager health = new HealthStatusManager();
+    ServerCredentials serverCredentials =
+        credentialsType == CredentialsType.XDS
+            ? XdsServerCredentials.create(InsecureServerCredentials.create())
+            : InsecureServerCredentials.create();
+    // Since the main server may be using TLS, we start a second server just for plaintext health
+    // checks
+    int healthPort = port + 1;
```

Comment:
Is there documentation that configures the health check to actually use this new port?

Also, forcing the use of port+1 seems a bit surprising. Why not add a new flag to let the user pick the health port explicitly when using secure credentials?

Reply:
Again I am going to cite the example-xds. I am not sure the flag to specify the health-port will be that useful - most people  might be okay to use port + 1 as the health port

Reply:
> Is there documentation that configures the health check to actually use this new port?
> 

The example-xds doc is here https://cloud.devsite.corp.google.com/traffic-director/docs/security-proxyless-setup#creating_the_health_check_firewall_rule_and_backend_service which says "The xDS helloworld example server uses PORT_NUMBER + 1 as the plaintext health checking port."

Our documentation will be similar.